server/graphql/v2/object/PaymentMethod.js

```diff
@@ -62,13 +62,42 @@ export const PaymentMethod = new GraphQLObjectType({
             return pick(paymentMethod.data, ['email']);
           }
 
+          // Return HostCollectiveId for prepaid
+          if (paymentMethod.type === 'prepaid') {
+            return pick(paymentMethod.data, ['HostCollectiveId']);
+          }
```

Comment:
I think we should expose that in `limitedToHostCollectiveIds` / `limitedToHosts`. Would be more consistent and better structured.